account/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+
+class Account(models.Model):
+    user_account = models.CharField(max_length=30, null=False)
+    password     = models.CharField(max_length=70)
+    mobile       = models.CharField(max_length=20)
+    email        = models.CharField(max_length=50, null=False)
```

Comment:
```suggestion
    email        = models.EmailField(max_length=100, null=False, unique=True)
```
이메일은 고유하니까 unique 해주세요.